antarest/study/storage/variantstudy/model/command/update_xpansion_security_criterion.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2025, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+# Copyright (c) 2025, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+from typing import List, Optional
+
+from typing_extensions import override
+
+from antarest.study.business.model.xpansion_model import (
+    XpansionSecurityCriterionUpdate,
+    update_xpansion_security_criterion,
+)
+from antarest.study.dao.api.study_dao import StudyDao
+from antarest.study.storage.variantstudy.model.command.common import CommandName, CommandOutput, command_succeeded
+from antarest.study.storage.variantstudy.model.command.icommand import ICommand
+from antarest.study.storage.variantstudy.model.command_listener.command_listener import ICommandListener
+from antarest.study.storage.variantstudy.model.model import CommandDTO
+
+
+class UpdateXpansionSecurityCriterion(ICommand):
+    """
+    Command used to update xpansion security criterion
+    """
+
+    # Overloaded metadata
+    # ===================
+
+    command_name: CommandName = CommandName.UPDATE_XPANSION_SECURITY_CRITERION
+
+    # Command parameters
+    # ==================
+
+    criterion: XpansionSecurityCriterionUpdate
```

Comment:
See other comment, I would propose to simplify this and have a replace (put) instead of a real update (patch)